mlflow/tracking/_tracking_service/utils.py

```diff
@@ -53,6 +55,7 @@ def set_tracking_uri(uri: str) -> None:
                   Databricks CLI
                   `profile <https://github.com/databricks/databricks-cli#installation>`_,
                   "databricks://<profileName>".
+                - A :py:class:`pathlib.Path` instance
```

Comment:
@cacharle Looks like our docs CI is indicating that this reference cannot be resolved:

```
/home/circleci/project/mlflow/__init__.py:docstring of mlflow.tracking._tracking_service.utils.set_tracking_uri:: WARNING: py:class reference target not found: pathlib.Path
/home/circleci/project/mlflow/__init__.py:docstring of mlflow.tracking._tracking_service.utils.set_tracking_uri:13: WARNING: py:class reference target not found: pathlib.Path
/home/circleci/project/mlflow/tracking/__init__.py:docstring of mlflow.tracking._tracking_service.utils.set_tracking_uri:: WARNING: py:class reference target not found: pathlib.Path
/home/circleci/project/mlflow/tracking/__init__.py:docstring of mlflow.tracking._tracking_service.utils.set_tracking_uri:13: WARNING: py:class reference target not found: pathlib.Path
```

Can you try adjusting the reference? You can test locally by running `cd docs && make rsthtml` from the repository root.